### ISSUE TITLE
Ceph Grafana Dashboard - Total Objects

### ISSUE THREAD

**Issue by distributorofpain** [ceph/ceph-ansible#6032]
**Is your feature request related to a problem? Please describe.**
Cant see the total objects in the cluster, it appears the ceph node exporter doesnt include ceph_cluster_total_objects anymore.

**Describe the solution you'd like**
have ceph_cluster_total_objects added back.

**Describe alternatives you've considered**
Doesnt seem to be another type of counter, but i suppose a node exporter could be created just for this :/

**Additional context**
Image of an older dashboard trying to use the existing datasource created by ceph-ansible.
![image](https://user-images.githubusercontent.com/18275482/99178597-c97fea80-26e2-11eb-8da2-3a117ebc2e6f.png)

**Comment by tchaikov** [comment]
@distributorofpain hi Brent, i am not familiar with Grafana. but it seems that we do have this field. see https://github.com/ceph/ceph/blob/70828cb1a8014f8c92cf86014114fd7aab6861c9/monitoring/grafana/dashboards/pool-detail.json#L537-L544

**Comment by jerryrelmore** [comment]
Resurrecting a necro thread but providing an explanation for anyone else that may come across it: 

We've got this issue as well on an older production cluster which has a sibling cluster that's a couple of years even older. I was going through some old graphs earlier today performing high-level review in preparation for cluster upgrades in the next couple of quarters and saw that the oldest production cluster graph - that scrapes `ceph_cluster_total_objects` - was displaying fine, while the newer cluster using the same graphs had a blank display. I manually pulled the metrics to confirm that the endpoint was working on the newer cluster, which it was, but the returned metrics lacked `ceph_cluster_total_objects`.

The change appears to have been introduced when the `OBJECTS` column was dropped in commit `[342f309](https://github.com/ceph/ceph/commit/342f309645df886fb96eb401634e38376553e6d9)` from version 14.0.1 to 14.1.0.
